src/test/java/org/terasology/module/health/BlockTest.java

```diff
@@ -28,7 +28,7 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @ExtendWith(MTEExtension.class)
-@Dependencies("Health")
+@IntegrationEnvironment(dependencies = { "Health" })
 @Tag("MteTest")
```

Comment:
As mentioned in https://github.com/MovingBlocks/Terasology/issues/5087
> the class annotation `@IntegrationEnvironment` [...] will apply `MTEExtension`, mark it with an `MteTest` JUnit tag

Accordingly, you can remove the `@ExtendWith` and `@Tag` annotations as well.

Same for all other changes